### PR TITLE
Update StaleIssueCommentGenerator.php

### PR DESCRIPTION
**Pull request by 94noni**
relates to https://github.com/symfony/symfony/issues/50778
just a raw idea like this
I agree with the [comment](https://github.com/symfony/symfony/issues/50778#issuecomment-2200191716), sometime people comment and would like a feature but perhaps dont know that this framework is community driven first

### CODE REVIEW

File: src/Service/StaleIssueCommentGenerator.php [hunk 1]
```diff
@@ -71,6 +71,9 @@ private function feature(): string
         return <<<TXT
 Thank you for this suggestion.
 There has not been a lot of activity here for a while. Would you still like to see this feature?
+[Every feature request may or may not be developed in future versions based on the community.](https://symfony.com/doc/current/contributing/community/releases.html#development)
```
Comment (Nyholm) [inline src/Service/StaleIssueCommentGenerator.php:74]: I see you took that sentence from the website. But it is a strange sentence.. Maybe we can make it simpler?

```suggestion
[Every feature is developed by the community.](https://symfony.com/doc/current/contributing/community/releases.html#development)
```

**Review by Nyholm — COMMENTED** [review]
Thank you. 

I appreciate this change.

File: src/Service/StaleIssueCommentGenerator.php (hunk 1 above)
Comment (94noni) [reply src/Service/StaleIssueCommentGenerator.php:74]: Agreed

**Review by Nyholm — APPROVED** [review]
Thank you

**Pull request merged**